addons/dev-helpers/build-local-container.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -o nounset -o pipefail -o errexit
```

Reply:
Given there are no commands that run without checking the result, I put this line after validating the input. Otherwise, if you provide no args to the script, it fails with '$1' not being initialized instead of outputing the usage of the script. We could make it so it works with that at the top, just didn't find it useful to do it before the script args where handled

Reply:
I think you will not have the issue with "$1" if you use same recipe as for `commit-local-container.sh`:

`name="${1:-}"`

Reply:
Done!